shared/metadata.go

```diff
@@ -4,6 +4,40 @@
 
 package shared
 
+import (
+	"net/http"
+	"path"
+	"sort"
+
+	"github.com/go-yaml/yaml"
+	log "github.com/sirupsen/logrus"
+	"github.com/web-platform-tests/wpt-metadata/util"
+)
+
+// MetadataFlag determines whether Metadata Information returns along
+// with a test result query request.
+// TODO(kyleju): Remove this flag once feauture is launched.
+const MetadataFlag = "metadataInfo"
+
+// MetadataResponse is a response to a wpt-metadata query.
+type MetadataResponse struct {
+	Response MetadataResults
```

Comment:
```suggestion
	Response MetadataResults `json:"response"`
```

Reply:
Actually, do we really need this extra level?

Reply:
yea we don't need it